platform-connectors/pkg/nodemetadata/config.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodemetadata
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	DefaultCacheSize        = 50
+	DefaultCacheTTL         = 1 * time.Hour
+	DefaultAPITimeout       = 2 * time.Second
+	DefaultQPS              = 5.0
+	DefaultBurst            = 10
+	DefaultMaxRetries       = 3
+	DefaultDecodeProviderID = true
+)
+
+// DefaultAllowedLabels defines the recommended set of node labels to include in health events.
+// These labels are carefully selected to provide maximum operational value while minimizing
+// metadata bloat and cache churn.
+var DefaultAllowedLabels = []string{
+	// Topology labels (critical for failure domain analysis)
+	"topology.kubernetes.io/zone",   // Current standard (K8s 1.17+)
+	"topology.kubernetes.io/region", // Current standard (K8s 1.17+)
+	"failure-domain.beta.kubernetes.io/zone",   // Legacy (deprecated but still common)
+	"failure-domain.beta.kubernetes.io/region", // Legacy (deprecated but still common)
+
+	// Instance type (critical for cost attribution & capacity planning)
+	"node.kubernetes.io/instance-type",   // Current standard
+	"beta.kubernetes.io/instance-type",   // Legacy
+
+	// GPU-specific labels (NVSentinel domain)
+	"nvidia.com/gpu.present",             // GPU node identification
+	"nvidia.com/gpu.deploy.dcgm",         // DCGM deployment topology
+	"nvidia.com/gpu.deploy.driver",       // Driver deployment topology
+	"nvidia.com/gpu.count",               // Number of GPUs
+	"nvsentinel.dgxc.nvidia.com/dcgm.version",     // DCGM version tracking
+	"nvsentinel.dgxc.nvidia.com/driver.installed", // Driver installation status
```

Comment:
We don't need these, can we instead pick the ones that have cuda/driver versions on them?

Reply:
Can you check the official GPU operator docs on their github repo and confirm the labels, please?